recipes/rcat/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "RCAT" %}
+{% set version = "v1.1" %}
```

Comment:
Version numbers shouldn't contain a `v`

Reply:
I also noticed the version inside the setuptools metadata is out of date: https://github.com/jplind79/RCAT/blob/master/setup.py#L8

Personally I like using `setuptools_scm` to avoid this problem.

Reply:
Ok, thanks for bringing this to our attention. We will update and also check out setuptools_scm.